packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -187,6 +187,41 @@ describe('Table', () => {
             expect(headerContent?.textContent).toEqual('foo');
         });
 
+        it('sets title when header text is ellipsized', async () => {
```

Comment:
Are there cases we need to think about when the header content includes icons? In that case we recommend that clients set their own title; do we need to ensure we don't accidentally remove it?

Reply:
I don't think so. They would be setting the title on the icon--we are setting the title on a span around the icon (and whatever else they have as content in their column element). I believe that if there is a title set on both the icon and the span, hovering over the icon should give the icon's tooltip, since it is the inner element.